starboard/android/shared/media_capabilities_cache.cc

```diff
@@ -627,19 +609,20 @@ MediaCapabilitiesCache::MediaCapabilitiesCache() {
 
 void MediaCapabilitiesCache::UpdateMediaCapabilities_Locked() {
   mutex_.DCheckAcquired();
-
-  if (supported_hdr_types_is_dirty_.exchange(false)) {
+  if (capabilities_is_dirty_.exchange(false)) {
```

Comment:
Just want to double check, since we are running this on the first query, how long is the delay?  A delay of 100ms is acceptable, but 1 second is probably not.

Reply:
LGTM.  We should consider using a thread for the initialization of the cache, but that can be a future optimization.